library-front/src/components/BookCard/BookCard.tsx

```diff
@@ -1,10 +1,10 @@
 import { FC } from 'react'
 import { AiOutlineEdit as EditIcon } from 'react-icons/ai'
 import { MdDeleteForever as DeleteIcon } from 'react-icons/md'
+import { useNavigate } from 'react-router-dom'
 import { toast } from 'react-toastify'
-import { useJwt } from '../../App'
+import bookCoverPlaceholder from '../../assets/book-cover-placeholder.png'
 import { isAdmin, isLibrarian } from '../../services/AuthService'
-import { getBook } from '../../services/BookService'
```

Comment:
Sto mi pokazuje sve fajlove kao izmenjene? i sve komitove ? a ne samo izmenu koju si napravio :/

Reply:
To se i ja pitam, ranije sam radio rebase na feature granama kada zaostaju za mainom pa je moguce da je do toga. Ne znam da li bi isto bilo i merge iz maina u feature da sam radio